src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigbar">
+      <form 
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"  
+        method="get" 
+        data-qa="big"
+      >
+        <div>
+          <img 
+            src="./images/Search.svg" 
+            class="loop" 
+            alt="search"
+          >
+          <input 
+            type="text" 
+            placeholder="Try &quot Los Angeles&quot" 
```

Comment:
```suggestion
            placeholder="Try &quotLos Angeles&quot" 
```

Reply:
hi! thank you for your review, however, my placeholders were properly coded, i guess, because after editing them in a way you recommended me, the outline was wrong. Regarding the 2 checklist point, as i understand, i need to exclude all the repetitive features of the inputs during the hover or focus processes?